wazuh/wazuh-dashboard-plugins
# Add agent columns to vulnerabilities inventory table

### Description
This pull request adds `agent.name` and `agent.id` columns to the Vulnerabilities / Inventory table. It also fixes the use-docked-side-nav custom hook to evaluate the width of the viewport, which fixes the way the full-screen feature of the table is rendered in smaller windows.
 
### Issues Resolved
#6350 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/7b870c5e-c308-45f7-bc51-5579a9a19870)


### Test
- Go to Vulnerabilities / Inventory
- Use the full-screen feature of the Inventory table with and without the docked side navigation menu
- Check the table has the columns:

| agent.id | agent.name | package.name | package.version | vulnerability.severity | vulnerability.id | vulnerability.score.version | vulnerability.score.base |
| --- | --- | --- | --- | --- | --- | --- | --- |

### Check List
